### DISCUSSION TITLE
How to upgrade?

### DISCUSSION THREAD

**Discussion by d00gatr0n** [Serene-Arc/bulk-downloader-for-reddit#492] [Q&A]
When the previous version 2.2 was released, I ran the command `python3 -m pip install bdfr --upgrade` and BDFR was immediately upgraded to the latest version.

Version 2.3 was recently released, but when I run the same command now its telling me 2.2 is the latest version...

Can anyone help me with what I’m doing wrong and how to update to 2.3?

**Comment by aliparlakci** [comment]
2.3 is not properly published, unfortunately. I will fix it today. Sorry for the inconvenience.

**Reply by d00gatr0n** [reply]
No problems, thanks for the reply. Just thought I was doing something wrong :)

**Reply by aliparlakci** [reply]
Should be fixed now, fyi.

**Reply by d00gatr0n** [reply]
Just tried it then, upgraded straight away. Thanks!
